twirssi.pl

```diff
@@ -1925,7 +1926,7 @@ sub get_updates_child {
                 unless (&get_timeline( $fh, $frusers[ $fix_replies_index{$username} ],
                                                $username, $twits{$username}, \%context_cache, $is_regular )) {
                     $error++;
-                    push @error_types, 'replies';
+                    push @error_types, 'Failed to get_timeline(frusers)';
```

Comment:
Dan Boger said:
> 
> In twirssi.pl:
> 
> > @@ -1925,7 +1926,7 @@ sub get_updates_child {
> >                  unless (&get_timeline( $fh, $frusers[ $fix_replies_index{
> $username} ],
> >                                                 $username, $twits{$usernam
> e}, \%context_cache, $is_regular )) {
> >                      $error++;
> > -                    push @error_types, 'replies';
> > +                    push @error_types, 'Failed to get_timeline(frusers)';
> 
> @frusers perhaps?

My thought was to make the different error messages distinct enough to figure 
out which error was triggered. Dunno if the contents of @frusers is useful in 
that error message. What do you think?


- Salve

-- 
#!/usr/bin/env perl
sub AUTOLOAD{$AUTOLOAD=~/.*::(\d+)/;seek(DATA,$1,0);print# Salve Joshua Nilsen
getc DATA}$"="'};&{'";@_=unpack("C*",unpack("u*",':50,$'.#    <sjn@foo.no>
'3!=0"59,6!`%%P\0!1)46%!F.Q`%01,`'."\n"));eval "&{'@_'}";  __END__ is near! :)